04.ls/ls.rb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+COL_NUM = 3
+GAP = 2
+
+target_path = !ARGV[0].nil? ? ARGV[0] : '.'
```

Comment:
最初に気づけてなかったですが、

```suggestion
target_path = ARGV[0].nil? ? '.' : ARGV[0]
```

のほうがシンプルになりそうです。